common/src/main/java/io/netty/util/AsciiString.java

```diff
@@ -551,6 +556,15 @@ public boolean contentEqualsIgnoreCase(CharSequence string) {
         return true;
     }
 
+    private boolean misalignedEqualsIgnoreCase(AsciiString rhs, byte[] value, byte[] rhsValue) {
+        for (int i = arrayOffset(), j = rhs.arrayOffset(), end = i + length(); i < end; ++i, ++j) {
+            if (!equalsIgnoreCase(value[i], rhsValue[j])) {
+                return false;
+            }
+        }
+        return true;
+    }
```

Comment:
Not terribly important but passing in the `rhs` and the `rhsValue` is a strange signature: maybe we can either pass in the offset or get the array from `rhs`?.

Reply:
@franz1981 please address @bryce-anderson comment